glean-core/src/metrics/url.rs

```diff
@@ -80,15 +80,15 @@ impl UrlMetric {
             return;
         }
 
-        let s = value.into();
+        let mut s = value.into();
         if s.len() > MAX_URL_LENGTH {
             let msg = format!(
                 "Value length {} exceeds maximum of {}",
                 s.len(),
                 MAX_URL_LENGTH
             );
             record_error(glean, &self.meta, ErrorType::InvalidOverflow, msg, None);
-            return;
+            s = s[..MAX_URL_LENGTH].to_string();
         }
```

Comment:
We have a whole function to make this easier and safer!

```suggestion
        let s = truncate_string_at_boundary_with_error(glean, &self.meta, value, MAX_URL_LENGTH);
```

Reply:
Fixed this in https://github.com/mozilla/glean/pull/2199/commits/e0eede33369df85e8a767ccb75a50ad3b513432d

Had to add import as well